qml_essentials/model.py

```diff
@@ -23,7 +23,7 @@ def __init__(
         n_layers: int,
         circuit_type: str,
         data_reupload: bool = True,
-        initialization: str = "random",
+        initialization: Union[str, Circuit] = "random",
```

Comment:
Is this needed? The initialisation type is still just a string, also for custom circuits, right?

Reply:
Uhm, looks like I successfully managed to mess up the two lines.. the updated type is of course meant for the `circuit_type` parameter not the initialization :see_no_evil: